tuf/src/interchange/cjson/shims.rs

```diff
@@ -8,23 +8,24 @@ use crate::error::Error;
 use crate::metadata::{self, Metadata};
 use crate::Result;
 
-const SPEC_VERSION: &str = "1.0";
+const SPEC_VERSION: &str = "1.0.0";
+
+// Ensure the given spec version matches our spec version.
+//
+// We also need to handle the literal "1.0" here, despite that fact that it is not a valid version
+// according to the SemVer spec, because it is already baked into some of the old roots.
+fn valid_spec_version(other: &str) -> bool {
+    other == SPEC_VERSION || other == "1.0"
+}
 
 fn parse_datetime(ts: &str) -> Result<DateTime<Utc>> {
```

Comment:
Thanks, yeah we should use `parse_from_3339(...)` here. I wish we had a formal one to fully parse ISO 8601 since that's what the spec says we could get, but this is at least better than what we currently have.

Could you add a test for this as well, just to make sure we support the old scheme, and what go-tuf can produce?